src/csharp/Extensions/Microsoft.Spark.Extensions.Delta.E2ETest/DeltaTableTests.cs

```diff
@@ -337,7 +337,9 @@ public void TestSignaturesV3_0_X()
             _spark.Range(15).Write().Format("delta").SaveAsTable(tableName);
 
             Assert.IsType<DeltaTable>(DeltaTable.ForName(tableName));
-            Assert.IsType<DeltaTable>(DeltaTable.ForName(_spark, tableName));
+            DeltaTable table = Assert.IsType<DeltaTable>(DeltaTable.ForName(_spark, tableName));
```

Comment:
`Assert.IsType<DeltaTable>` not required if you are setting it to `DeltaTable table`.